HaikuPorter/Port.py

```diff
@@ -711,14 +711,14 @@ def extractPatchset(self):
 		## use enumerate to avoid your own couters
 		for s, source in enumerate(self.sources, 1):
 			if s == 1:
-				patchSetFileName = self.name + '-' + self.version + '.patchset'
-				archPatchSetFileName = (self.name + '-' + self.version + '-'
+				patchSetFileName = self.baseName + '-' + self.version + '.patchset'
+				archPatchSetFileName = (self.baseName + '-' + self.version + '-'
```

Comment:
Is `archPatchSetFileName` supposed to be for architecture specific patches? I haven't actually looked at what this does exactly, but if that is the case, maybe the original name should stay for these?

In a recipe, `$secondaryArchSuffix` could be used to refer to them, if needed.

Reply:
A foo$secondaryArchSuffix.patchset isn't the way I think we should go, the intention should be to use the patchset over the architectures.
I'm not familiar with the `archPatchSetFileName` function, but did a check and it's working as expected.

Reply:
I meant to only refer to the second assignment here, the first line is fine, I think. I was just speculating what the second one could be for.

Reply:
Ah! my bad :) Maybe someone with some more insight can shed a light here?

Reply:
Checking `patchSetFileName` only results in the expected outcome, checking `archPatchSetFileName` only results in the _x86 suffix still for the patchset, so that shouldn't bee used.